app/models/user.rb

```diff
@@ -25,7 +25,8 @@ class User
 
   # Relationships
   has_and_belongs_to_many :patients, inverse_of: :users
-
+  belongs_to :pledge_generated_by, class_name: 'Patient', inverse_of: 'current_user'
```

Comment:
Fun fact about document stores: They really, really, really suck at relational stuff like this :) I think we can take this out until we have a specific use case for needing to see all pledges generated by a particular user, go ahead and revert this change, if you don't mind.